packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -582,6 +604,9 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 							onClick={(): void => {
 								setIndex(Number(rowId));
 								setSelectedEvent(item);
+								usageLogger?.sendTelemetryEvent({
+									eventName: `TelemetryEventClicked${item.eventName}`,
+								});
```

Comment:
In this case the `item.eventName` is unrelated to the telemetry event's name. Let's move it into a `details: { telemetryEventName: item.eventName }`. In general, the `eventName` we pass to the logger APIs should be very "static", and any differences between two logged events "of the same kind" should go in `details`.

That also applies to `MaxTelemetryEventSetTo${data.optionText}` for example.